charts/network-agent/values.yaml

```diff
@@ -0,0 +1,19 @@
+honeycomb:
```

Comment:
Do we want to add the debug and log level options or are we intentionally omitting them for the first pass?

Reply:
I was going to say omit for now, but they definitely add value and will likely be more useful during initial feedback. I'll add them 👍🏻

Reply:
I've added a simpler solution of allow extra env vars to be set and passed into the daemonset in https://github.com/honeycombio/helm-charts/pull/292/commits/c0545bde0c1debfec2c2d02e75a687453af14e17. Our and the OTel charts do the same thing.

I like this better because it doesn't let some accidentally to mess around with debug / development settings. They need to know what to set.